src/gluonts/model/trivial/mean.py

```diff
@@ -60,10 +61,20 @@ def __init__(
 
     def predict(self, dataset: Dataset, **kwargs) -> Iterator[SampleForecast]:
         for item in dataset:
-            mean = np.mean(item["target"][-self.context_length :])
+            if self.context_length is not None:
+                target = item["target"][-self.context_length :]
```

Comment:
I like this option

Reply:
Makes it easier to derive this Predictor from others :)